src/main/java/com/marek/onlinebookstore/service/order/OrderService.java

```diff
@@ -0,0 +1,20 @@
+package com.marek.onlinebookstore.service.order;
+
+import com.marek.onlinebookstore.dto.order.OrderDto;
+import com.marek.onlinebookstore.dto.order.OrderItemDto;
+import com.marek.onlinebookstore.dto.order.PlacingOrderRequestDto;
+import com.marek.onlinebookstore.model.User;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+
+public interface OrderService {
+    OrderDto makeOrder(PlacingOrderRequestDto requestDto);
+
+    List<OrderDto> getHistory(User user, Pageable pageable);
+
+    List<OrderItemDto> getFromOrder(Long orderId);
+
+    OrderItemDto getFromOrder(Long orderId, Long id);
+
+    OrderDto updateStatus(Long id);
```

Comment:
The method 'updateStatus' should take in an additional parameter to indicate the new status of the order. Currently, it's not clear how the status would be updated just based on the order id.